src/Package-Dev.appxmanifest

```diff
@@ -9,7 +23,7 @@
       </ProxyStub>
     </Extension>
   </Extensions>
-  <Identity Name="Microsoft.Windows.DevHome.Dev" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" Version="0.0.0.0" />
+  <Identity Name="Microsoft.Windows.DevHome.Dev" Publisher="CN=timkur" Version="0.0.1.0" />

```

Comment:
Revert

Reply:
Oops. What's the best way to build packages locally without all of this getting modified?

Reply:
.\build.ps1